packages/react-components/src/Concepts/Slots.stories.mdx

```diff
@@ -61,12 +61,12 @@ It also has an `icon` slot that only renders when there is neither an image nor
 
 A property of a component can be identified as a slot by its type.
 
-- IntrinsicShorthandProps: The props for a slot that supports one or more HTML element types, such as 'div'.
-- ObjectShorthandProps: The props for a slot that additionally supports a JSX component or component props object.
-- ShorthandRenderFunction: A render function for a slot.
+- IntrinsicSlotProps: The props for a slot that supports one or more HTML element types, such as 'div'.
+- ObjectSlotProps: The props for a slot that additionally supports a JSX component or component props object.
+- SlotRenderFunction: A render function for a slot.
 
-Notice these types contain the term 'shorthand'.
-Shorthand means the slot props can accept 'shorthand' data such as primitive values and props objects.
+In our composition system, slot props can accept 'shorthand' data such as primitive values and props objects, that they
+use to determine the overall.
```

Comment:
I think you accidentally a word from this sentence.

Reply:
Oops, fixed!